src/pages/accounting/index.tsx

```diff
@@ -1,166 +1,168 @@
+            <Typography component="div">{children}</Typography>
+          </Box>
+        )}
+      </div>
+    )
+  }
+
+  const AntTabs = withStyles({
+    root: {
+      borderBottom: '1px solid #e8e8e8',
+    },
+    indicator: {
+      backgroundColor: '#3f51b5',
+    },
+  })(Tabs)
+
+  interface StyledTabProps {
+    label: string
+  }
 
-      const AntTabs = withStyles({
-        root: {
-          borderBottom: '1px solid #e8e8e8',
+  const AntTab = withStyles((theme: Theme) =>
+    createStyles({
+      root: {
+        textTransform: 'none',
+        minWidth: 72,
+        backgroundColor: 'transparent',
+        fontWeight: theme.typography.fontWeightRegular,
+        marginRight: theme.spacing(4),
+        fontFamily: [
+          '-apple-system',
+          'BlinkMacSystemFont',
+          '"Segoe UI"',
+          'Roboto',
+          '"Helvetica Neue"',
+          'Arial',
+          'sans-serif',
+          '"Apple Color Emoji"',
+          '"Segoe UI Emoji"',
+          '"Segoe UI Symbol"',
+        ].join(','),
+        '&:hover': {
+          color: '#3f51b5',
+          opacity: 1,
         },
-        indicator: {
-          backgroundColor: '#3f51b5',
+        '&$selected': {
+          color: '#3f51b5',
+          fontWeight: theme.typography.fontWeightMedium,
         },
-      })(Tabs);
-
-      interface StyledTabProps {
-        label: string;
-      }
-      
-      const AntTab = withStyles((theme: Theme) =>
-        createStyles({
-          root: {
-            textTransform: 'none',
-            minWidth: 72,
-            backgroundColor: 'transparent',
-            fontWeight: theme.typography.fontWeightRegular,
-            marginRight: theme.spacing(4),
-            fontFamily: [
-              '-apple-system',
-              'BlinkMacSystemFont',
-              '"Segoe UI"',
-              'Roboto',
-              '"Helvetica Neue"',
-              'Arial',
-              'sans-serif',
-              '"Apple Color Emoji"',
-              '"Segoe UI Emoji"',
-              '"Segoe UI Symbol"',
-            ].join(','),
-            '&:hover': {
-              color: '#3f51b5',
-              opacity: 1,
-            },
-            '&$selected': {
-              color: '#3f51b5',
-              fontWeight: theme.typography.fontWeightMedium,
-            },
-            '&:focus': {
-              color: '#3f51b5',
-            },
-          },
-          selected: {},
-        }),
-      )((props: StyledTabProps) => <Tab disableRipple {...props} />);
+        '&:focus': {
+          color: '#3f51b5',
+        },
+      },
+      selected: {},
+    }),
+  )((props: StyledTabProps) => <Tab disableRipple {...props} />)
 
-    
-    return (
-        <div>
-          {props.nodeHealth?.status === 'ok' && props.health ?
+  return (
+    <div>
+      {
+        // FIXME: this should be broken up
+        /* eslint-disable no-nested-ternary */
+        nodeHealth?.status === 'ok' && health ? (
```

Comment:
I find it's better to extract logic in a variable (e.g. `isNodeHealthy = ...`) and in the JSX only rely on such variables instead of mixing rendering with logic.